### ISSUE TITLE
Only download saves for installed games

### ISSUE THREAD

**Issue by ZeptoBST** [mtkennerly/ludusavi#430]
### What's your idea?

So when I download my saves from the cloud onto my ROG Ally, it also downloads saves for games I only have installed on my main PC and my Laptop, thus wasting bandwith and storage on my Ally.

It would be great to have an option to only download saves for games that appear on the "Backup Mode" page and "Custom Games" page.

**Comment by pktiuk** [comment]
I agree, but I would rephrase this enhancement as: `Allow cloud sync only for selected games`.
